src/main/scala/devices/i2c/I2C.scala

```diff
@@ -540,8 +540,7 @@ trait HasI2CModuleContents extends MultiIOModule with HasRegMap {
                                                                  nextCmd := data
                                                              }
                                                              true.B
-                                                }
-                                                )))
+                                                }), RegFieldDesc("cmd_status","I2C tx and rx command status")))
```

Comment:
should be `volatile=true`

Reply:
I think a better description may be "On write, update command. On Read, report status."